src/web_interface/components/component_base.py

```diff
@@ -46,7 +47,9 @@ def __call__(self, view_function: Callable) -> Callable:
 
 
 class ComponentBase:
-    def __init__(self, app, db: FrontendDatabase, intercom, status: RedisStatusInterface, api=None):  # noqa: PLR0913
+    def __init__(  # noqa: PLR0913
+        self, app, db: FrontendDatabase, intercom: InterComFrontEndBinding, status: RedisStatusInterface, api=None
```

Comment:
I personally dislike this style of line breaking and would prefer to have one argument on one line.

Reply:
changed